.github/workflows/build_steps.yml

```
@@ -163,8 +163,8 @@ jobs:
       - name: Discover test directory names
         if: inputs.job_type == 'follower'
         # There are so few nonreg tests, run them in the hypothesis runner
-        run: find python/tests/* -maxdepth 0 -type d ! -regex '.*\(__pycache__\|util\|nonreg\)' -printf '"%f",' |
-              sed 's/^/test_dirs=[/ ; s/"hypothesis"/"{hypothesis,nonreg}"/ ; s/,$/]/' | tee -a $GITHUB_ENV
+        run: find python/tests/* -maxdepth 0 -type d ! -regex '.*\(__pycache__\|util\|nonreg\|scripts\)' -printf '"%f",' |
+              sed 's/^/test_dirs=[/ ; s/"hypothesis"/"{hypothesis,nonreg,scripts}"/ ; s/,$/]/' | tee -a $GITHUB_ENV
```

Nit. Is there a reason not to use something simpler like:
```suggestion
        run: echo 'test_dirs=["{hypothesis,nonreg,scripts}","integration","stress","unit"]' | tee -a $GITHUB_ENV
```

?

---

If new folder under `test` folder is added, with your suggestion, won't it require some additional changes here, which is not necessary with the my commit?

---

Yes this nitpick was about having something more explicit, which can be modified if a folder is added.

Anyway, let's merge this PR so that we can merge the others ones for `4.0.x`.